### ISSUE TITLE
ESP8285 support

### ISSUE THREAD

**Issue by RM-FPV** [zs6buj/MavlinkToPassthru#78]
Thanks a lot for the great project!
Can be ESP8285 support also added? If it's already supported are there any changes in setup of FW needed?

**Comment by zs6buj** [comment]
Hi RM-FPV,

I would need to get an ESP8285 board to test, but I guess it would be relatively easy to add it in as a variant.

**Comment by RM-FPV** [comment]
Hi zs6buj!
This is a nice module, if you are looking for the one
https://aliexpress.com/item/32877575176.html

**Comment by zs6buj** [comment]
Hi, that particular offering on ali wanted $48 for shipping to my home base, so I ordered [this one](https://www.banggood.com/DT-06-Wireless-WiFi-Serial-Transmissions-Module-TTL-to-WiFi-Compatible-HC-06-Bluetooth-External-Antenna-Version-Optional-p-1424145.html?cur_warehouse=CN&ID=566572&rmmds=search](url)) instead. I'm in another country until the end of April, so I'm hoping the board gets to me before I leave for home.

**Comment by RM-FPV** [comment]
Hi, the module you found looks even better because of external antenna!

One more question please - when transferring mavlink to wifi, will it be also possible to connect notebook to this module to use some groundstation SW for displaying plane position on a map?

**Comment by zs6buj** [comment]
Yes you can connect several wifi clients, like gcs or tracker or whatever else.

**Comment by RM-FPV** [comment]
That's really great!
Thanks a lot!
Just waiting for 8285 version.

**Comment by RM-FPV** [comment]
Hi zs6buj!
Have you received your modules? Would be nice to know any news about the FW.
Thanks.

**Comment by zs6buj** [comment]
Hi RM-FPV, yes I did receive a module, but I unfortunately have very little free time right now. I'll earmark this issue and get back to it as soon as I can.

**Comment by RM-FPV** [comment]
Got it! Thank you for reply!

**Comment by zs6buj** [comment]
Closing as stale
